### PR TITLE
solution

### PR DESCRIPTION
**Pull request by LiliiaDanylenko**
- [DEMO LINK](https://LiliiaDanylenko.github.io/js_employees_table_DOM/)

### CODE REVIEW

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]
test

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
test

**Review by vlad-demchuk — APPROVED** [review]

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
test

**Review by Bohdan-Maksymiuk — APPROVED** [review]

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,283 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('thead tr');
+const tableBody = document.querySelector('tbody');
+let temp = null;
```
Comment (vlad-demchuk) [inline src/scripts/main.js:5]: use more descriptive variable names

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,283 @@
...
+  if (findIndex > 2) {
+    input.type = 'number';
+  }
+
+  if (findIndex === 2) {
+    input.remove();
+    selectedCell.append(select);
+
+    select.innerHTML = `
+      ${offices.map(office => `<option>${office}</option>`).join('')}
```
Comment (vlad-demchuk) [inline src/scripts/main.js:239]: you do it twice
`${offices.map(office => `<option>${office}</option>`).join('')}`

you can create a separate function to to it

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,283 @@
...
+    if (ev.code === 'Enter') {
+      input.blur();
+
+      pushNotification(
+        10, 'success', 'The data has been successfully changed', 'success'
+      );
+    }
+  });
+});
+// for test
```
Comment (vlad-demchuk) [inline src/scripts/main.js:283]: remove comments

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
GJ

I can change data in table to invalid one
image

**Review by vlad-demchuk — APPROVED** [review]

**Pull request closed without merging**